components/d2l-filter-dropdown/d2l-filter-dropdown-option.js

```diff
@@ -0,0 +1,66 @@
+/**
+`d2l-filter-dropdown-option`
+Polymer-based web component for checkbox menu items.
+
+@demo demo/checkbox-menu.html checkbox menu
```

Comment:
Can remove this line (used to be for auto-generated Polymer docs) and the `polymer-modulizer` comment below